docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+
+## Campaign
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines during the creation of campaign. Campaign ID is a reference to the campaign in the applications, services and data connections.
+
+There are two types of campaigns:
+
+- **Anonymous** - engages with the audience, increases the advertising impressions, or collects cookies of personalized audiences.
+- **Identified** - gathers information on the user by filling out an identification form.
+
+## Publication channel
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign:
+
+- **Widget/iframe** - uses a JavaScript code or an HTML iframe from Qualifio. Additionally, you can use oEmbed, Facebook Instant articles, and AMP technologies to publish iframes. You need to manually paste the code into your website or CMS.
+
+- **Minisite** - uses a unique URL from Qualifio that can hosts the campaign. This URL points to your subdomain.
+
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
+    By default, every type of publication channel is responsive and compatible with all mobile devices.
+
+## Work with campaigns
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
+To access the Qualifio Back Office page and create or manage campaigns, click the **Manage campaigns in Qualifio** button under the **Ibexa Engage** menu.
+To display your campaign on the website, use one of the available tools - Campaign block or Campaign custom tag.
+For more information about creating and managing campaigns, see [Qualifio Engage documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
+
+### Campaign block
+
+To promote your campaign, you can add [Campaign block](../content_management/block_reference.md#campaign-block) in Page Builder.
+In the **Campaign** block you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
+
+![Campaign block](img/campaign_block.png "Campaign block")
+
+### Campaign custom tag
+
+You can embed campaign in the Rich Text field with Campaign custom tag.
```

Comment:
```suggestion
You can embed a campaign within the [Rich Text field](../content_management/create_edit_content_items.md#edit-rich-text-fields) by using a Campaign custom tag.
```
Question: Should we use "custom tag" here? Editors do not know this concept. For them it is jus one more button in the toolbar. Therefore I'd use "option".
On another occasion I'd also fix the SeenThis! description in the linked article.

Reply:
"Custom tag" - this is the feature name.